README.md

```diff
@@ -1,1527 +1,22 @@
-# Mozilla Foundation JavaScript Style Guide() {
+# Mozilla Foundation JS Style Guide
 
-*Originally forked from the Airbnb style guide -- a mostly reasonable approach to JavaScript*
+This repo contains "RC" files, which represent our standard rules, for several different JavaScript linters ([JSHint](http://jshint.com/), [JSCS](http://jscs.info/), and [JSBeautifier](https://github.com/beautify-web/js-beautify)).
 
+In your project you can use npm to install the RC files as a module, which can then be individually used by your project's build system (eg: Gulp or Grunt). Most linting tasks/plugins allow you to specify an external RC file, which you can point to your copies in `node_modules`.
 
-## Table of Contents
+Simply run `npm install mofo-style --save-dev`.
 
-  1. [Types](#types)
-  1. [undefined](#undefined)
-  1. [Objects](#objects)
-  1. [Arrays](#arrays)
-  1. [Strings](#strings)
-  1. [Functions](#functions)
-  1. [Properties](#properties)
-  1. [Variables](#variables)
-  1. [Hoisting](#hoisting)
-  1. [Conditional Expressions & Equality](#conditional-expressions--equality)
-  1. [Blocks](#blocks)
-  1. [Comments](#comments)
-  1. [Whitespace](#whitespace)
-  1. [Commas](#commas)
-  1. [Semicolons](#semicolons)
-  1. [Type Casting & Coercion](#type-casting--coercion)
-  1. [Naming Conventions](#naming-conventions)
-  1. [Accessors](#accessors)
-  1. [Constructors](#constructors)
-  1. [Events](#events)
-  1. [Modules](#modules)
-  1. [jQuery](#jquery)
-  1. [ECMAScript 5 Compatibility](#ecmascript-5-compatibility)
-  1. [Testing](#testing)
-  1. [Performance](#performance)
-  1. [Resources](#resources)
-  1. [In the Wild](#in-the-wild)
-  1. [Translation](#translation)
-  1. [The JavaScript Style Guide Guide](#the-javascript-style-guide-guide)
-  1. [Contributors](#contributors)
-  1. [License](#license)
+## Task Naming
 
-## Types
+You should have at least 2 style tasks in your runner.
 
-  - **Primitives**: When you access a primitive type you work directly on its value
+- `validate` - Run all 3 linters in read only mode and report any errors.
+- `clean` - Run all 3 linters but also run JSBeautifier in write mode to automatically fix any formatting errors that it's capable of repairing.
 
-    + `string`
-    + `number`
-    + `boolean`
-    + `null`
-    + `undefined`
+## Rules
 
-    ```javascript
-    var foo = 1;
-    var bar = foo;
+Refer to the [RC files](https://github.com/MozillaFoundation/javascript-style-guide/tree/master/linters) for specifics.
```

Comment:
I figure at least while things are in flux this will keep things from getting out of sync...